api/desecapi/models.py

```diff
@@ -897,20 +897,36 @@ def _act(self):
         self.domain.save(update_fields=['renewal_state', 'renewal_changed'])
 
 
-def captcha_default_content():
-    alphabet = (string.ascii_uppercase + string.digits).translate({ord(c): None for c in 'IO0'})
-    content = ''.join([secrets.choice(alphabet) for _ in range(5)])
+def captcha_default_content(kind: str) -> str:
```

Comment:
Now that this is not used as a default for the field, but depends on `.kind`, it should be method of the `Captcha` class

Reply:
fixed

Reply:
moving this code requires fiddling with the initial migration. I say let's keep it where it is.

Reply:
ah yes, that makes sense. Sorry for the ill-posed suggestion!